vm/src/hint_processor/builtin_hint_processor/bootloader/bootloader_hints.rs

```diff
@@ -285,6 +288,63 @@ pub fn save_packed_outputs(exec_scopes: &mut ExecutionScopes) -> Result<(), Hint
     Ok(())
 }
 
+/// Implements
+/// const COMPUTE_FACT_TOPOLOGIES: &str = "from typing import List
```

Comment:
I think we leave out `const FOO: &str =` by convention, at least I have been

Reply:
Yes, my bad.